packages/dashboard-core-plugins/src/panels/PanelContextMenu.tsx

```diff
@@ -34,6 +41,11 @@ class PanelContextMenu extends PureComponent<
     return [...(glContainer?.tab?.header?.tabs ?? [])];
   }
 
+  handleReopenLast(): void {
```

Comment:
Should actually be re-opening the last panel in this _stack_, which means we also need to store which stack the panel was closed. I think pull that functionality into a separate PR, leave this PR just for the global re-open last panel action (which will show up as a keyboard shortcut under Settings). 

We also will want a context menu on dashboards, but dashboards in community do not have context menu items (yet), could be another ticket as well.

This was after our discussion with @dsmmcken yesterday.